pkg/scheduler/internal/cache/cache.go

```diff
@@ -741,6 +741,7 @@ func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 		ps, ok := cache.podStates[key]
 		if !ok {
 			klog.Fatal("Key found in assumed set but not in podStates. Potentially a logical error.")
+			continue
```

Comment:
`Klog.Fatal` would cause program to exit, so `continue` is not needed.

Reply:
thanks.